### ISSUE TITLE
[Moderate] Links opens new window without warning

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#608]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87757 |
| **Checkpoint** | 3.2 Predictable › Context Changes (on Input) (3.2.2.a) |
| **Impact** | Moderate |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 02 Import/Export Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/12/26 at 1:10 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Best Practice |
| **Reported By** | kim.casey@ablr360.com |

## Description

The link text does not indicate that the link opens in a new window. (AAA requirement 3.2.5)
In Export Settings, select New, then navigate to the Setup Questions section, select the Help button. It opens a internet browser in another window

## Recommendation

Rule: 
If a link triggers the page to open in a new window or tab, the user should be provided a warning ahead of time.

How To Fix: 
Fix this issue by informing users ahead of time of such behavior by methods such as the text label for the UI control or some advisory text placed before the control that cautions the user of this behavior.
Employ ONE of the following:
1. Include the warning in the text describing a control - Shop now (opens in a new window)
2. Use CSS to provide a warning before opening a new window (see reference #1 below.

References: 
Giving Users Advanced notice when opening in a new window: https://www.w3.org/WAI/WCAG21/Techniques/general/G201
Opening new windows only when necessary: https://www.w3.org/WAI/WCAG21/Techniques/general/G200
Using the Target Attribute to open in a new window: https://www.w3.org/WAI/WCAG21/Techniques/html/H83

Background: 
Opening new windows automatically when a link is activated can be disorienting for people who have difficulty perceiving visual content, and for some people with cognitive disabilities, if they are not warned in advance.  Providing a warning allows the user to decide it they want to leave the current window, and the warning will help them find their way back, if they do decide they would like to go to the new window. It will help them understand that the "back" button will not work and that they have to return to the last window they had open, in order to find their previous location.

## Screenshots

![02_open in new window](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87757/02_open_in_new_window.png)
